.github/workflows/ode-toolbox-build.yml

```diff
@@ -0,0 +1,67 @@
+name: ODE-toolbox build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        with_gsl: ["0", "1"]
+        sympy_version: ["==1.4", ""]
+
+    steps:
+      - name: Checkout ODE-toolbox code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install apt dependencies
+        run: |
+          sudo apt update
+          sudo apt install pkg-config python3-all-dev
+
+      - name: Install GSL
+        run: |
+          cd ..
+          wget http://ftp.wrz.de/pub/gnu/gsl/gsl-2.5.tar.gz
+          tar -xzf gsl-2.5.tar.gz
+          cd gsl-2.5
+          ./configure && make && sudo make install
+          cd ..
+          cd ode-toolbox
+          export LD_LIBRARY_PATH=/usr/local/lib
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip pytest pycodestyle codecov pytest-cov
+          python -m pip install -r requirements.txt
+          if [ "${{ matrix.with_gsl }}" == "1" ]; then wget "https://files.pythonhosted.org/packages/17/fc/9c11163f017a0d6877141d4d4edb619b0f05990d865caab840a2d0463581/pygsl-2.3.0.1.tar.gz" && mv -v pygsl-2.3.0.1.tar.gz pygsl-2.3.0.tar.gz && python -m pip install pygsl-2.3.0.tar.gz ; fi
+          export PYTHON_VERSION=`python -c "import sys; print('.'.join(map(str, [sys.version_info.major, sys.version_info.minor])))"`
+          echo "Python version detected:"
+          echo $PYTHON_VERSION
+
+          pip uninstall --yes sympy
+          pip install sympy${{ matrix.sympy_version }}
+          echo "Using sympy version:"
+          python -c "import sympy; print(sympy.__version__)"
+
+      - name: Install ODE-toolbox
+        run: |
+          echo PYTHONPATH=`pwd` >> $GITHUB_ENV
+          python setup.py install
+
+      - name: Static code style analysis
+        run: |
+          python3 -m pycodestyle $GITHUB_WORKSPACE -v --ignore=E241,E501,E303,E714,E713,E714,E252 --exclude=$GITHUB_WORKSPACE/doc,$GITHUB_WORKSPACE/.eggs,$GITHUB_WORKSPACE/build,$GITHUB_WORKSPACE/.git,$GITHUB_WORKSPACE/odetoolbox.egg-info,$GITHUB_WORKSPACE/dist
+
+      - name: Run unit tests
+        run: |
+          python -m pytest -s -o log_cli=true -o log_cli_level="DEBUG" --cov=./ tests
+
+      - name: Run codecov
+        run: |
+          codecov
```

Comment:
Would this report be displayed anywhere?

Reply:
It ought to still show up as a comment on the PR by the codecov bot (see https://github.com/nest/ode-toolbox/pull/46#issuecomment-813656953 for an example), and should be visible via the coverage button in the docs header (see https://ode-toolbox.readthedocs.org/).